helper/testhelpers/testhelpers.go

```diff
@@ -954,10 +954,10 @@ func SetupLoginMFATOTP(t testing.T, client *api.Client) (*api.Client, string, st
 	// Configure a default TOTP method
 	totpConfig := map[string]interface{}{
 		"issuer":                  "yCorp",
-		"period":                  5,
+		"period":                  20,
 		"algorithm":               "SHA256",
 		"digits":                  6,
-		"skew":                    0,
+		"skew":                    1,
```

Comment:
The MFA login command test introduced with the MFA tokenhelper fix has been flaky since it was introduced: https://app.circleci.com/pipelines/github/hashicorp/vault/44169/workflows/f5a1340a-316a-4b11-947b-7c4be34a8d39/jobs/536590

These failures are fairly frequent. The skew shouldn't really matter too much, but it seemed like a setting of `1` would be more robust, since it will allow validation against the last TOTP value in the event of a timeout (if I'm not mistaken).